src/a4/domain/Board.java

```diff
@@ -1,5 +1,71 @@
 package a4.domain;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class Board {
+	List<BoardSpace> spaces = new ArrayList<BoardSpace>();
+
```

Comment:
Oh good point... It doesn't have to be constant I guess jaja I feel like while I was talking to Chancey he said it might be useful to have a count of the board spaces? But I guess it works to just get the list of board spaces and get the count of that so never mind!